Tasks/PackerBuildV0/src/utilities.ts

```diff
@@ -84,6 +84,17 @@ export function readJsonFile(filePath: string): string {
     return content;
 }
 
+export function createTemplateVarFile (templateVariables: Map<string, string>): string {
+    let filePath: string = path.resolve(tl.getVariable('Agent.TempDirectory'), Math.random().toString(36).replace('0.', '') + '.json');
```

Comment:
Instead of random, you can also use Date.now() which is good enough

Reply:
I tried using Date.now(), but since we create 2 separate var files one after the other, there were name collisions a few times. That is why I am using to Math.random.

Reply:
ok